### ISSUE TITLE
if location disabled in settings it ask to enable location in settings but can't get location after enable it.

### ISSUE THREAD

**Issue by vicky-droid** [BirjuVachhani/locus-android#65]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by vicky-droid** [comment]
steps to reproduce the issue:

1. disable location in settings.
2. click to fetch location from app
3. it ask permission to enable location.
4. it ask to enable the location in settings. enable it.
5. now again try to fetch the location.
6. not getting location.

**Comment by character-not-found** [comment]
I am facing the same issue, with the same steps

**Comment by BirjuVachhani** [comment]
Fix available in v4.1.0

```groovy
implementation 'com.github.BirjuVachhani:locus-android:4.1.0'
```
